assets/css/style.css

```diff
@@ -178,7 +178,19 @@ h2 {
     color: #ccc;
 }
 
+.txt-type {
+    color: rgba(255, 255, 0, 0.8);
+}

```

Comment:
```suggestion
.txt-type {
    color: rgba(255, 255, 0, 0.8);
}
```

```
.txt-type::after {
    color: #fff;
    content: "_";
    animation: animate-cursor 1s infinite;
}
```

try using these colors #FFA726, #FFC107 and #FF5722 
same color for both text and cursor. Choose any of the three. Whichever looks good.

Reply:
@Rocker2102